src/components/BookCard/style.css

```diff
@@ -0,0 +1,56 @@
+.book-container{
```

Comment:
naming each style file the same is a bad practice. Ideally name them according to their respective component. For eg: This file should be named as BookCardStyles.css